src/schema/v2/schema.ts

```diff
@@ -187,8 +188,7 @@ const rootFields = {
   channel,
   city: City,
   cities,
-  notificationPreferences,
-  // collection: Collection,
+  conversationsConnection: Conversations,
```

Comment:
I'm cool with the new root field. I _think_ it may not be needed (vs. using `conversationsConnection` under `Partner` and under `me`, for instance), but that wouldn't support admin use case at all, and while possibly premature, this does seem like an appropriate root-field addition even if the existing partner/me could be reused. 👍